src/main/java/org/smartregister/pathevaluator/dao/TaskDao.java

```diff
@@ -47,4 +47,6 @@
 	org.smartregister.domain.Task updateTask(org.smartregister.domain.Task task);
 
 	List<Task> findTasksByJurisdiction(String jurisdiction, String planIdentifier);
+
+	List<Task> findTasksByJurisdiction(String jurisdiction);
```

Comment:
Can we use the existing method `List<Task> findTasksByJurisdiction(String jurisdiction, String planIdentifier);` by passing `null` in the place of plan Identifier and modify its implementation on the server-side; add criteria of plan Identifier only if it is not null.

Reply:
If it takes longer to refactor this, then I am good with your approach as well.

Reply:
That's a good idea. I believe it will currently take more time since this is already implemented on the server and client

Reply:
What do you think?

Reply:
Yes, please use the existing method and modify its implementation on the server-side.

Reply:
Cool